client/packages/invoices/src/Prescriptions/DetailView/Payments/PaymentsModal.tsx

```diff
@@ -172,7 +164,7 @@ export const PaymentsModal: FC<PaymentsModalProps> = ({
               label={t('label.discount-rate')}
               Input={
                 <BasicTextInput
-                  value={`${discountRate}%`}
+                  value={`${discountPercentage}%`}
```

Comment:
This is pretty nitpicky, but I don't think it should show the "%" symbol twice. I.e. should either be "Discount: 10%", or "Discount(%): 10".
![Screenshot 2025-02-28 at 10 49 27 AM](https://github.com/user-attachments/assets/7a47e2ea-0f85-4e8d-acb9-d1924aa6c51b)

Reply:
Yeah I'm with you...

Reply:
[e61ad1f](https://github.com/msupply-foundation/open-msupply/pull/6741/commits/e61ad1f85e70a1ee2d8c4bfd62bbe12d650b56f7)